.github/workflows/main.yaml

```
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
```

Usually I'm using the following:

```suggestion
on: [push, pull_request]
```

Maybe that's the reason why the build was not triggered on this pull request?